dm/tests/many_tables/run.sh

```diff
@@ -120,6 +153,8 @@ cleanup_process
 
 run $*
 
+trap - EXIT
```

Comment:
before we reset the trap, we should manually run `restore_timezone` so it will be run once at any case